diagnostic.md

```diff
@@ -100,11 +128,22 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --request PATCH \
+  -header "Content-Type: application/json" \
+  http://www.example.com/donuts/42\
+  --data '{
+    "name": "Krüller"
+  }'
 ```
```

Comment:
I'm not positive this would work with your syntax changes. It may, but this is a preferred convention.

```sh
curl --include --request "PATCH" "http://example.com/donuts/42" \
  --header "Content-Type: application/json" \
  --data '{
    "donut": {
      "name": "Krüller",
    }
  }'
```